functions/ldap_access_order_defaults.pp

```diff
@@ -0,0 +1,29 @@
+# Generate the proper ldap_access_order defaults based on the version of SSSD
+# in place
+#
+# @return [Array[String]]
+#
+function sssd::ldap_access_order_defaults {
+  $_defaults = [
+    'expire',
+    'lockout'
+  ]
+
+  if $facts['sssd_version'] {
+    if versioncmp($facts['sssd_version'], '1.14.0') >= 0 {
+      $_result = [
+        'ppolicy',
+        'pwd_expire_policy_reject',
```

Comment:
Based on the descriptions of pwd_expire_policy_reject, pwd_expire_policy_warn, and pwd_expire_policy_renew, it only makes sense to me that one of these options is used in the ldap_access_order.  As it is written, pwd_expire_policy_reject will be applied.

Reply:
I'll drop it back to just `reject`